### DISCUSSION TITLE
In progress status

### DISCUSSION THREAD

**Discussion by snehadudhankar** [github/india#413] [Students]
Actually I have done Azure Fundamentals part 1: Describe core Azure concepts but in my futurereadytalent dashboard shows in progress status. Can anyone know how it's solve?

**Comment by 19C91A0503** [comment]
It takes 1 week to update.

**Reply by snehadudhankar** [reply]
Okay ! Thank you

**Reply by TheSuranaverse** [reply]
A group for all to get notified for any `Official FRT Announcement`, `Discuss & Clear any Doubts`, `Learn & Complete` together.
One-stop place for daily `Events/Workshops` + `Hackathons` + `Internships/Jobs` + `Free Scholarship, Courses, Swags & Goodies` + `24/7 Live Chat and Mentors to help you with your Doubts`  + `24/7 Live Music` & much more...

Currently, we are helping all the `Future Ready Talent` Interns clear their doubts & queries on a personal basis either on chat or voice, whichever they prefer. We will also be assisting them on Projects & help them complete this internship 🤗

Link to join: https://discord.gg/drQ7ySCdX2
GitHub Solution Forum: https://github.com/github/india/discussions/353

Thank you 🙏

**Comment by Shreyas95033** [comment]
I have uploaded my project along with video stating service used but they are saying that service use is not justified kindly resubmit. Please anyone help what to do ?
![Screenshot_20220214-130208_Gmail](https://user-images.githubusercontent.com/90888322/153819474-539bf998-d838-4256-acb2-033332b7b596.jpg)
I have uploaded a screenahot please looke it and tell me what to do.
